### PR TITLE
2nd project

### PR DESCRIPTION
**Pull request by wizardfan**
will some inherent unit test failure

### CODE REVIEW

File: src/app/services/api-data.service.ts [hunk 1]
```diff
@@ -40,6 +40,29 @@ export class ApiDataService {
...
+          checksum: entry['_source']['checksum'],
+          checksumMethod: entry['_source']['checksumMethod']
+          })
+        );
+      }),
+      retry(3),
+      catchError(this.handleError),
+    );
+  }
+
```
Comment (Alexey-ebi) [inline src/app/services/api-data.service.ts]: Overall this method is fine, just to be more consistent with other methods can you:
1. remove headers (it's not necessary here, as Angular returns json by default)
2. provide interface for returning data
map((data: any) => {
        return data.hits.hits.map( entry => ({
          name: entry['_source']['name'],
          fileId: entry['_id'],
          experiment: entry['_source']['experiment']['accession'],
          assayType: entry['_source']['experiment']['assayType'],
          experimentTarget: entry['_source']['experiment']['target'],
          run: entry['_source']['run']['accession'],
          readableSize: entry['_source']['readableSize'],
          checksum: entry['_source']['checksum'],
          checksumMethod: entry['_source']['checksumMethod']
          } as FileForProjectsTable )

export interface FileForProjectsTable {
  name: string;
  fileId: string;
  experiment: string;
  assayType: string;
  experimentTarget: string;
  run: string;
  readableSize: string;
  checksum: string;
  checksumMethod: string;
}

File: src/app/services/api-data.service.ts [hunk 2]
```diff
@@ -85,6 +108,27 @@ export class ApiDataService {
     );
   }
 
+  getAllOrganismsFromProject(project: string) {
+    const url = this.hostSetting.host + 'organism/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```
Comment (Alexey-ebi) [inline src/app/services/api-data.service.ts]: same thing here, headers is not required.

File: src/app/services/api-data.service.ts [hunk 3]
```diff
@@ -101,6 +145,29 @@ export class ApiDataService {
     );
   }
 
+  getAllSpecimensForProject(project: string) {
+    const url = this.hostSetting.host + 'specimen/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```
Comment (Alexey-ebi) [inline src/app/services/api-data.service.ts]: same thing here, headers is not required.

File: src/app/services/api-data.service.ts [hunk 4]
```diff
@@ -244,6 +311,25 @@ export class ApiDataService {
     );
   }
 
+  getAllArticlesForProject(project: string) {
+    const url = this.hostSetting.host + 'article/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```
Comment (Alexey-ebi) [inline src/app/services/api-data.service.ts]: same thing here, headers is not required.

File: src/app/services/api-data.service.ts (hunk 1 above)
Comment (wizardfan) [reply src/app/services/api-data.service.ts]: done in c257a2a

File: src/app/services/api-data.service.ts (hunk 2 above)
Comment (wizardfan) [reply src/app/services/api-data.service.ts]: done in c257a2a

File: src/app/services/api-data.service.ts (hunk 3 above)
Comment (wizardfan) [reply src/app/services/api-data.service.ts]: done in c257a2a

File: src/app/services/api-data.service.ts (hunk 4 above)
Comment (wizardfan) [reply src/app/services/api-data.service.ts]: done in c257a2a

Comment (Alexey-ebi) [reply src/app/services/api-data.service.ts]: getAllArticlesForProject still has headers option in request, can you remove it please

**Pull request merged**